lib/msal-browser/src/error/BrowserAuthError.ts

```diff
@@ -4,672 +4,337 @@
+        desc: BrowserAuthErrorMessages[
+            BrowserAuthErrorCodes.databaseUnavailable
+        ],
     },
     unableToAcquireTokenFromNativePlatform: {
-        code: "unable_to_acquire_token_from_native_platform",
-        desc: "Unable to acquire token from native platform. For a list of possible reasons visit aka.ms/msaljs/browser-errors.",
+        code: BrowserAuthErrorCodes.unableToAcquireTokenFromNativePlatform,
+        desc: BrowserAuthErrorMessages[
+            BrowserAuthErrorCodes.unableToAcquireTokenFromNativePlatform
+        ],
     },
     nativeHandshakeTimeout: {
-        code: "native_handshake_timeout",
-        desc: "Timed out while attempting to establish connection to browser extension",
+        code: BrowserAuthErrorCodes.nativeHandshakeTimeout,
+        desc: BrowserAuthErrorMessages[
+            BrowserAuthErrorCodes.nativeHandshakeTimeout
+        ],
     },
     nativeExtensionNotInstalled: {
-        code: "native_extension_not_installed",
-        desc: "Native extension is not installed. If you think this is a mistake call the initialize function.",
+        code: BrowserAuthErrorCodes.nativeExtensionNotInstalled,
+        desc: BrowserAuthErrorMessages[
+            BrowserAuthErrorCodes.nativeExtensionNotInstalled
+        ],
     },
     nativeConnectionNotEstablished: {
-        code: "native_connection_not_established",
-        desc: "Connection to native platform has not been established. Please install a compatible browser extension and run initialize(). For more please visit aka.ms/msaljs/browser-errors.",
+        code: BrowserAuthErrorCodes.nativeConnectionNotEstablished,
+        desc: BrowserAuthErrorMessages[
+            BrowserAuthErrorCodes.nativeConnectionNotEstablished
+        ],
     },
     uninitializedPublicClientApplication: {
-        code: "uninitialized_public_client_application",
-        desc: "You must call and await the initialize function before attempting to call any other MSAL API. For more please visit aka.ms/msaljs/browser-errors.",
+        code: BrowserAuthErrorCodes.uninitializedPublicClientApplication,
+        desc: BrowserAuthErrorMessages[
+            BrowserAuthErrorCodes.uninitializedPublicClientApplication
+        ],
     },
     nativePromptNotSupported: {
-        code: "native_prompt_not_supported",
-        desc: "The provided prompt is not supported by the native platform. This request should be routed to the web based flow.",
+        code: BrowserAuthErrorCodes.nativePromptNotSupported,
+        desc: BrowserAuthErrorMessages[
+            BrowserAuthErrorCodes.nativePromptNotSupported
+        ],
     },
 };
 
 /**
  * Browser library error class thrown by the MSAL.js library for SPAs
  */
 export class BrowserAuthError extends AuthError {
-    constructor(errorCode: string, errorMessage?: string) {
-        super(errorCode, errorMessage);
+    constructor(errorCode: string) {
+        super(errorCode, BrowserAuthErrorMessages[errorCode]);
```

Comment:
@tnorling I have some test utilities in our app to make sure we're receiving and handling MSAL errors in a consistent manner. Should the second arg to `super` be `BrowserAuthErrorMessages[errorCode].desc` here instead? I noticed this when `BrowserAuthError` started emitting differently than the other errors after being converted to string. 

![image](https://github.com/AzureAD/microsoft-authentication-library-for-js/assets/26070760/4cad82d0-f0dd-4418-a85a-127b218817bf)

Reply:
Sorry, overlooked the `s` at the end of `BrowserAuthErrorMessages`